File: ympa_fsharp_giraffe_server/src/model/GenerateGoodsRealizationReportRequest.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic

module GenerateGoodsRealizationReportRequest =

  //#region GenerateGoodsRealizationReportRequest


  type GenerateGoodsRealizationReportRequest = {
    CampaignId : int64;
    Year : int;
    Month : int;
  }
  //#endregion

File: src/gatling/scala/org/openapitools/client/model/GenerateGoodsRealizationReportRequest.scala

package org.openapitools.client.model


case class GenerateGoodsRealizationReportRequest (
    /* Идентификатор кампании. */
    _campaignId: Long,
    /* Год. */
    _year: Integer,
    /* Номер месяца. */
    _month: Integer
)
object GenerateGoodsRealizationReportRequest {
    def toStringBody(var_campaignId: Object, var_year: Object, var_month: Object) =
        s"""
        | {
        | "campaignId":$var_campaignId,"year":$var_year,"month":$var_month
        | }
        """.stripMargin
}
